server/constants.js

```diff
@@ -1,9 +1,12 @@
 /* @flow */
 
 export const HOST = {
-    LOCALHOST:     'localhost.paypal.com',
-    PAYPAL:        '.paypal.com',
-    PAYPALOBJECTS: 'www.paypalobjects.com'
+    LOCALHOST:      'localhost.paypal.com',
+    PAYPAL:         '.paypal.com',
+    PAYPALOBJECTS:  'www.paypalobjects.com',
+    LOCALTUNNEL:    'paypal.loca.lt',
```

Comment:
Can you share more details about `paypal.loca.lt`? Could we technically be attacked if someone go a hold of this domain on [localtunnel](https://theboroer.github.io/localtunnel-www/)?

Reply:
@gregjopa For sure!  These local url's are only evaluated when `env=local`.  This is our unified way of being able to test SPB changes locally on native devices.

Reply:
@mnicpt thanks for explaining. So as long as we have the `__ENV__ === ENV.LOCAL` check for these urls, we prevent any security vulnerabilities. That makes sense.